### ISSUE TITLE
transaction 'tax' attribute changed or missing?

### ISSUE THREAD

**Issue by gpetty** [AuthorizeNet/sdk-python#60]
This line worked a few weeks ago (with version 1.0.9):
 tax  = transactionDetailsResponse.transaction.tax

Now (v. 1.0.11), it generates the following error:
  File "src/lxml/lxml.objectify.pyx", line 229, in lxml.objectify.ObjectifiedElement.__getattr__ (src/lxml/lxml.objectify.c:3835)
  File "src/lxml/lxml.objectify.pyx", line 450, in lxml.objectify._lookupChildOrRaise (src/lxml/lxml.objectify.c:6539)
AttributeError: no such child: {AnetApi/xml/v1/schema/AnetApiSchema.xsd}tax

Other attributes of transactionDetailsResponse.transaction appear to still be present. The API guide still shows the following attributes as being valid:

* tax 	Contains information about any taxes.

** amount 	Amount of the tax.

** name 	Name of the tax.

** description 	Description of the tax.

**Comment by gpetty** [comment]
More generally, it seems that quite a bit of my working code broke with the update from 1.0.9 to 1.0.11.   Is there a changes log that explains exactly what changed and what I need to do to efficiently get my code working again?

**Comment by brianmc** [comment]
Hi @gpetty, first of all apologies for not documenting this SDK version breaking change, we have updated the release notes at https://github.com/AuthorizeNet/sdk-python/releases/tag/1.0.10 

The SDK has been in some flux as we had no real choice but to move away from pyxb for response handling as it was a strict schema XML library which could not accommodate our server-side API changes.  We acknowledge the fallout from that change is not trivial but we're hoping the SDK design time breaking change is a lesser of two evils over a runtime breaking change.  Overall I think we moved the SDK out of beta a little prematurely but I can assure you we're fully committed to production support of the Python SDK.

We have updated our getTransactionDetails sample to demonstrate the usage for optional elements such as tax.  https://github.com/AuthorizeNet/sample-code-python/blob/master/TransactionReporting/get-transaction-details.py
